.travis.yml

```diff
@@ -76,8 +76,7 @@ before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.* ]]; then export CODECOVERAGE=1 ; phpdbg -qrr vendor/bin/phpunit --coverage-clover=clover.xml; fi
```

Comment:
The coverage build is segfaulting which to me indicates an issue with xdebug and or php7.